parsers/verilog.c

```diff
@@ -556,6 +557,19 @@ static void initializeVerilog (const langType language)
 	buildKeywordHash (language, IDX_VERILOG);
 	addKeywordGroup (&verilogKeywords, language);
 	addKeywordGroup (&verilogDirectives, language);
+	if (tagContents == NULL)
+		tagContents = ptrArrayNew ((ptrArrayDeleteFunc)deleteToken);
+
+}
+
+static void finalizeVerilog (const langType language,
```

Comment:
I think we don't need finalizeVerilog() and finalizeSystemVerilog().

I am not a Purist who frees everything before `exit()`.
If we add this, we should also free the hash table entries added by `addKeyword()`.